src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {
+  CSSTransition,
+  TransitionGroup,
+} from 'react-transition-group';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../Todo/TodoItem';
+
+interface Props {
+  todos: Todo[];
+  tempTodo: Todo | null;
+  todosInProcess: number[];
+  deleteTodo: (todo: Todo) => void;
+}
+
+export const TodoList: React.FC<Props> = React.memo(({
+  tempTodo,
+  todos,
+  todosInProcess,
+  deleteTodo,
+}) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings